### PR TITLE
Fix static analysis issues and code quality in PerfLab load.php files

### PR DESCRIPTION
**Pull request by westonruter**
## Summary

I noticed a few issues in the `load.php` and `admin/load.php` files, mainly identified with a stricter level of static analysis in PHPStan (#775) and/or PhpStorm's static analysis. There are other issues in the other PHP files, but I just wanted to limit the scope to these two files (and a JS file). The following issues are fixed:

* The use of `esc_js()` is brittle and should be replaced with `wp_json_encode()`.
* Using `var` in JS is obsolete in favor of `const`/`let`.
* Lacking PHP7 typing on function parameters and return values.
* LanguageTool is complaining about grammar in a couple places.
* Most substantively, the `deactivate_plugins()` function was being used as if it returned a value. It returns `void`.

### CODE REVIEW

File: admin/load.php [hunk 1]
```diff
@@ -131,8 +131,8 @@ function perflab_render_pointer( $pointer_id = 'perflab-admin-pointer', $args =
 	<script id="<?php echo esc_attr( $pointer_id ); ?>" type="text/javascript">
 		jQuery( function() {
 			// Pointer Options.
-			var options = {
-				content: '<h3><?php echo esc_js( $args['heading'] ); ?></h3><p><?php echo wp_kses( $args['content'], $wp_kses_options ); ?></p>',
+			const options = {
+				content: <?php echo wp_json_encode( '<h3>' . esc_html( $args['heading'] ) . '</h3><p>' . wp_kses( $args['content'], $wp_kses_options ) . '</p>' ); ?>,
```
Comment (felixarntz) [inline admin/load.php]: Why is `wp_json_encode()` better here? It isn't really JSON, so this feels a bit weird to me semantically.

File: admin/load.php [hunk 2]
```diff
@@ -73,7 +73,7 @@ function perflab_render_settings_page() {
  *
  * @param string $hook_suffix The current admin page.
  */
-function perflab_admin_pointer( $hook_suffix ) {
+function perflab_admin_pointer( string $hook_suffix ) {
```
Comment (felixarntz) [inline admin/load.php]: Using PHP7 type hinting is a bit of a personal preference. I am personally in favor, but we haven't been doing it for the most part, so I feel this shouldn't be dictated by a tool.

I am not strictly opposed to the change, but it feels out of scope here and IMO should rather be the result of a conscious decision that the team is aware of.

File: admin/load.php (hunk 1 above)
Comment (westonruter) [reply admin/load.php]: Well, a few reasons:

1. It avoids the author having to keep track of the open and closing quotation marks, and remember whether they are single or double.
2. It consolidates the echoes into one rather than splitting it into two.

This also fixes a couple issues:

1. There is no escaping of the heading being done, so it's adding an `esc_html()`. If using `esc_js()` then it should probably should have been `esc_js( esc_html( $args['heading'] ) )`
2. Similarly, the `wp_kses( $args['content'], $wp_kses_options )` should have also been wrapped in `esc_js()` here in case the `content` contains some single quote mark or backslashes or some other character needing escaping.

By using `wp_json_encode()`, the code is made more robust against bugs being introduced.

File: admin/load.php (hunk 2 above)
Comment (westonruter) [reply admin/load.php]: ok, I reverted in 2269c6017335c995800d28c0a8948e5c921e8559

**Review by felixarntz — APPROVED** [review]

**Review by mukeshpanchal27 — APPROVED** [review]
